estimagic/dashboard/monitoring_app.py

```diff
@@ -73,60 +90,58 @@ def monitoring_app(
 
     doc.add_root(tabs)
 
+    if start_immediately:
+        activation_button = doc.get_model_by_name("activation_button")
+        activation_button.active = True
+
 
-def _get_group_to_params_from_database(database):
-    """Map each group name to the parameters' names that belong to it.
+def _create_id_column(df):
+    """Create a column that gives the position for plotted parameters and is None else.
 
     Args:
-        database (sqlalchemy.MetaData): Bound metadata object.
+        df (pd.DataFrame)
 
     Returns:
-        group_to_params (dict): keys are the group names, values are parameter names.
+        ids (pd.Series): integer position in the DataFrame unless the group was
+            None, False, np.nan or an empty string.
 
     """
-    optimization_problem = read_last_rows(
-        database=database,
-        table_name="optimization_problem",
-        n_rows=1,
-        return_type="dict_of_lists",
-    )
-    start_params = optimization_problem["params"][0]
-    group_to_params = _map_groups_to_params(start_params)
-    return start_params, group_to_params
+    ids = pd.Series(range(len(df)), dtype=object, index=df.index)
+    ids[df["group"].isin([None, False, np.nan, ""])] = None
+    return ids.astype(str)
 
 
-def _map_groups_to_params(params):
-    """Map the group name to the ColumnDataSource friendly parameter names.
+def _map_groups_to_param_values(params, column_name):
+    """Map the group name to lists of one column's values of the groups parameters.
 
     Args:
-        params (pd.DataFrame):
-            DataFrame with the parameter values and additional information such as the
-            "group" column and Index.
+        params (pd.DataFrame): Includes the "group" and "id" columns.
+        column_name (str): name of the column for which to return the parameter values.
 
     Returns:
-        group_to_params (dict):
-            Keys are the values of the "group" column. The values are lists with
-            bokeh friendly strings of the index tuples identifying the parameters
-            that belong to this group. Parameters where group is None, "" or False
-            are ignored.
+        group_to_values (dict): Keys are the values of the "group" column.
+            The values are lists of parameter values of the parameters belonging
+            to the particular group.
 
     """
-    group_to_params = {}
-    for group in params["group"].unique():
-        if group is not None and group == group and group != "" and group is not False:
-            group_to_params[group] = list(params[params["group"] == group]["name"])
-    return group_to_params
+    to_plot = params[~params["group"].isin([None, False, np.nan, ""])]
+    group_to_inds = to_plot.groupby("group").groups
```

Comment:
what does `_inds` stand for?

Reply:
indices. I changed it to group_to_indices in a675a68.